src/OSSupport/File.cpp

```diff
@@ -453,6 +453,29 @@ AString cFile::ReadWholeFile(const AString & a_FileName)
 
 
 
+AString cFile::ReplaceFileNameInvalidChars(const AString & a_FileName, char a_Replacement)
+{
+	AString res(a_FileName);
+	for (auto & ch: res)
+	{
+		switch (ch)
+		{
+			case ':':
+			case '\\':
```

Comment:
If you want to go for the minimum cross platform set you also need to exclude everything other than A-Z. 
For example their would be an issue with IPv4 addresses on FAT32, mounted on a system that doesn't long filenames.